src/Api/Clients/OrderClient.cs

```diff
@@ -12,24 +12,43 @@ public interface IOrderClient
         Task<ApiResponse<IEnumerable<Order>>> GetOpenOrdersAsync();
         Task<ApiResponse<IEnumerable<Guid>>> CancelOpenOrdersAsync(string productId = null);
         Task<ApiResponse<Order>> PlaceOrderAsync(string side, string productId, decimal size, decimal price, string type, string cancelAfter = null, string timeInForce = null);
+        Task<ApiResponse<Order>> PlaceOrderAsync(string side, string productId, decimal size, decimal price, string type, bool postOnly, string cancelAfter = null, string timeInForce = null);
     }
 
-    public class OrderClient : GdaxClient
+	public class OrderClient : GdaxClient
```

Comment:
- The indenting is now incorrect
- Why doesn't OrderClient implement IOrderClient?